app/components/school-curriculum-inventory-institution-manager.js

```diff
@@ -0,0 +1,141 @@
+import Component from '@ember/component';
+import { isPresent } from '@ember/utils';
+import { inject as service } from '@ember/service';
+import { validator, buildValidations } from 'ember-cp-validations';
+import ValidationErrorDisplay from 'ilios-common/mixins/validation-error-display';
+
+const Validations = buildValidations({
+  name: [
+    validator('presence', {
+      presence: true
+    }),
+    validator('length', {
+      max: 100
+    }),
+  ],
+  aamcCode: [
+    validator('number', {
+      allowString: true,
+      integer: true,
+      gte: 1,
+      lte: 99999
+    })
+  ],
+  addressStreet: [
+    validator('presence', {
+      presence: true
+    }),
+    validator('length', {
+      max: 100
+    }),
+  ],
+  addressCity: [
+    validator('presence', {
+      presence: true
+    }),
+    validator('length', {
+      max: 100
+    }),
+  ],
+  addressStateOrProvince: [
+    validator('presence', {
+      presence: true
+    }),
+    validator('length', {
+      max: 50
+    }),
+  ],
+  addressZipCode: [
+    validator('presence', {
+      presence: true
+    }),
+    validator('length', {
+      max: 10
+    }),
+  ],
+  addressCountryCode: [
+    validator('presence', {
+      presence: true
+    }),
+    validator('length', {
+      max: 2
+    }),
+  ],
+});
+
+export default Component.extend(ValidationErrorDisplay, Validations, {
+  store: service(),
+  "data-test-school-curriculum-inventory-institution-manager": true,
+  classNames: ["school-curriculum-inventory-institution-manager"],
+  isSaving: false,
+  name: null,
+  aamcCode: null,
+  addressStreet: null,
+  addressCity: null,
+  addressStateOrProvince: null,
+  addressZipCode: null,
+  addressCountryCode: null,
+
+  didReceiveAttrs(){
+    this._super(...arguments);
+    if (isPresent(this.institution)) {
+      this.setProperties({
+        "name": this.institution.get("name"),
```

Comment:
Can just be `name: this.institution.get("name"),`. Quotes aren't necessary.

Reply:
handled.